homeassistant/components/sql/__init__.py

```diff
@@ -40,23 +39,12 @@
     DOMAIN,
     PLATFORMS,
 )
-from .util import redact_credentials
+from .services import async_setup_services
+from .util import redact_credentials, validate_sql_select
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def validate_sql_select(value: str) -> str:
```

Comment:
Could we move these to `util.py` in a preliminary PR?
Makes this PR easier and also would help on the other PR's with the rebasing.

Reply:
#153135